wren-ui/src/apollo/server/resolvers/modelResolver.ts

```diff
@@ -507,6 +508,24 @@ export class ModelResolver {
     return true;
   }
 
+  public async previewModelData(_root: any, args: any, ctx: IContext) {
+    const modelId = args.where.id;
+    const model = await ctx.modelRepository.findOneBy({ id: modelId });
+    if (!model) {
+      throw new Error('Model not found');
+    }
+
+    // pass the current mdl to wren engine to preview data, prevent the model is not deployed
+    const { manifest } = await ctx.mdlService.makeCurrentModelMDL();
+    const sql = `select * from ${model.referenceName}`;
+    const data = await ctx.wrenEngineAdaptor.previewData(
+      sql,
+      PREVIEW_MAX_OUTPUT_ROW,
```

Comment:
cc @andreashimin & @fredalai , only 100 rows will return to frontend for model preview as well